src/TokenApiClient.cs

```diff
@@ -9,31 +9,31 @@
 
 namespace Laserfiche.Oauth.Api.Client
 {
-    public class ClientCredentialsHandler : IClientCredentialsHandler
+    public class TokenApiClient : ITokenApiClient
```

Comment:
Should this be renamed to something else instead of TokenApiClient? Since the name doesn't imply this client is for the client credentials flow

Reply:
Fixed